src/main.rs

```diff
@@ -25,6 +25,11 @@ fn main() {
         process::exit(1);
     }
 
+    if args[0].eq("--version") {
+        println!("Europa Lang v0.0.0");
+        process::exit(1);
```

Comment:
Why do you exit with code 1?

Reply:
(most programs will exit normally when you run --version)

![image](https://user-images.githubusercontent.com/61561933/125627606-caa32595-2fca-4f58-a855-b973f03b7131.png)

Reply:
let's also add the `-v` flag, common in windows cli.

Reply:
@cursorweb Would it be "-v" or "-V"? Either one can be used to mean "version" or "verbose", and there isn't too much consensus on it.

Reply:
I've just noticed this doesn't actually work either. `argv[0]` is the program name, and you've also asserted that the amount of arguments must be 2. Should we bring in a argument parser?

Reply:
That was just meant to be an example

Reply:
@DynamicSquid What? So why did you create a pull request for it?

Reply:
Well, the main focus of the PR was the tips - I never expected you to merge it or anything

Reply:
@DynamicSquid Well then create an issue? PR's are for merging code, don't make one if you don't want it to be merged...

Reply:
But I was planning to edit some code to show you how to implement tip, but it turns out most of it was just explaining. But that doesn't, let's focus on the new Issue instead :)